src/dotnet/commands/dotnet-install/dotnet-install-tool/InstallToolCommand.cs

```diff
@@ -14,16 +14,25 @@
 
 namespace Microsoft.DotNet.Tools.Install.Tool
 {
-    public class InstallToolCommand : CommandBase
+    internal class InstallToolCommand : CommandBase
     {
+        private static IToolPackageObtainer _toolPackageObtainer;
```

Comment:
You made this static but you are not really using it as such.

Reply:
oops, this should be readonly

Reply:
fixed